app/controllers/admin/stats_controller.rb

```diff
@@ -0,0 +1,70 @@
+# -*- encoding : utf-8 -*-
+
+class Admin::StatsController < ApplicationController
+  def index
+    # total user count
+    @users_count = User.count
+
+    # The most recent users to join
+    @users_stats = params.fetch(:users_stats, {})
+    @recent_users = recent_users
+
+    # Query Solr for top depositors
+    @active_users = top_depositors
+
+    # Count of documents by permissions
+    @files_count = document_by_permission
+
+    # Query Solr for top file formats
+    @top_formats = top_formats
+
+    render 'index'
+  end
+
+  private 
+  def top_depositors_count
+    count = params[:dep_count].to_i 
+    count.in?(5..20) ? count : 5
+  end
+
+  def document_by_permission
+    # make sure everything is up to date in solr
+    ActiveFedora::SolrService.instance.conn.commit("expungeDeletes"=>true)
```

Comment:
Question everything @flyingzumwalt does ;)


> On Jun 9, 2015, at 12:29 PM, cam156 <notifications@github.com> wrote:
> 
> In app/controllers/admin/stats_controller.rb:
> 
> > +
> > +    # Query Solr for top file formats
> > +    @top_formats = top_formats
> > +
> > +    render 'index'
> > +  end
> > +
> > +  private 
> > +  def top_depositors_count
> > +    count = params[:dep_count].to_i 
> > +    count.in?(5..20) ? count : 5
> > +  end
> > +
> > +  def document_by_permission
> > +    # make sure everything is up to date in solr
> > +    ActiveFedora::SolrService.instance.conn.commit("expungeDeletes"=>true)
> @jcoyne I was copying the code from scholarsphere. It came from this commit psu-stewardship/scholarsphere@b3a531c#diff-eb2a74120fa8e97a8ce630c4a345ac0a
> 
> Not really sure if it can be removed, but since you believe it can I will see if the test still passes.
> 
> —
> Reply to this email directly or view it on GitHub.
>